kornia/augmentation/augmentations.py

```diff
@@ -1,21 +1,34 @@
-from typing import Callable, Tuple, List, Union, cast
+from typing import Callable, Tuple, Union, cast
 
 import torch
 import torch.nn as nn
-from torch.distributions import Uniform
 
-from kornia.geometry.transform.flips import hflip, vflip
-from kornia.color.adjust import AdjustBrightness, AdjustContrast, AdjustSaturation, AdjustHue
-from kornia.color.gray import rgb_to_grayscale
-from kornia.geometry import pi
+from kornia.augmentation.functional import *
```

Comment:
I tend  to import what I really need, otherwise could be difficult to track used functions below in the code. Import as F or just the ones you need. Besides, if we want to behave with similar torch philosophy, we might abstract to another augmentation.utils/params module all the non exposed functions